my-app/src/test/java/com/mycompany/app/AppTest.java

```diff
@@ -0,0 +1,32 @@
+package com.mycompany.app;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+
+public class AppTest 
+{
+    @Test
+	void testQ_rsqurtNormalInput() {
+		float a = 16;
+		float actual= findInverseSquareRoot.Q_rsqurt(a);
+        float expected = 0.25f;
```

Comment:
put in a comment, something like  '1/sqrt(16) is 0.25f'. Otherwise people wonder what this is for.

Reply:
Done